### ISSUE TITLE
collectd repository for ubuntu bionic

### ISSUE THREAD

**Issue by kszarlej** [collectd/collectd#3128]
https://pkg.ci.collectd.org/deb/dists/ currently doesn't have packages built for ubuntu bionic. 
official ubuntu repo for bionic (ubuntu universe) has older 5.7 version. Can we expect to add newest stable 5.8 collectd built for ubuntu bionic in https://pkg.ci.collectd.org/deb/dists/?

**Comment by sunkuranganath** [comment]
Thanks for posting this question @kszarlej. Looking forward for a reply. 
It would be great if anyone could provide the process to coordinate with collectd package distribution with Ubuntu.

**Comment by mrunge** [comment]
Usually, providing builds for distributions is a distribution thing. For example, @rubenk updates and builds packages for Fedora or EPEL.

Looking at pkg.ci.collectd.org, it looks like all builds have stopped somewhere in 2018.

**Comment by PhilippWendler** [comment]
https://pkg.ci.collectd.org is listed as the official source of Debian/Ubuntu packages on the [download page](https://collectd.org/download.shtml), yet all new versions of collectd are missing.

It would be great if you could fix that service and start providing recent collectd packages again. It was very convenient for us to get newer packages from there than our distribution provides.

If not, please at least remove it from the documentation.

**Comment by faxm0dem** [comment]
I totally agree with @PhilippWendler. @mfournier could you please share your thoughts with us on your availability regarding maintenance of ci ? I'd be happy to help, or even take the responsibility of the task.

**Comment by Tampa** [comment]
I have been using the xenial version all the way up to jammy now without much problem beyond apt complaining about the mismatched releases. Have done some .deb packaging in the past, so if there is an old pipeline for that it can probably be updated to include the newer versions. Both github and jenkins can be made to build and package them as well, so if there is interest in offering official up to date packages via that repo again I'd be willing to fiddle with that.

I can't find much on why that repo isn't being updated anymore, but I suspect the extra work of packaging might be part of the reason. Some distros do have newer versions in their own repos, but can also lag behind somewhat, so an official source that offers development versions or simply the latest release is definitely appreciated :)
